### PR TITLE
Feature 44 sale details view

### PR DESCRIPTION
**Pull request by dimitrov93**

### CODE REVIEW

File: src/components/Sale/SaleParticipantsInfo.vue
[the remark this answers is not in the document]
Comment (dimitrov93) [reply src/components/Sale/SaleParticipantsInfo.vue:1]: fixed

File: src/components/Table/SalesTable.vue [hunk 1]
```diff
@@ -24,7 +24,13 @@
         hide-details
       ></v-text-field>
     </v-card-title>
-    <v-data-table :headers="tableColumns" :items="sales" :search="search">
+    <v-data-table
+      :headers="tableColumns"
+      :items="sales"
+      :search="search"
+      @click:row="rowHandler"
```
[the remark this answers is not in the document]
Comment (dimitrov93) [reply src/components/Table/SalesTable.vue:31]: fixed

File: src/router/index.js [hunk 2]
```diff
@@ -115,6 +116,12 @@ const routes = [
     component: SalesAdd,
     meta: { title: "New Sale", requiresAuth: true },
   },
+  {
+    path: "/sales/:saleId",
+    name: "Sales-Details",
+    component: SalesDetails,
```
[the remark this answers is not in the document]
Comment (dimitrov93) [reply src/router/index.js:122]: fixed

File: src/components/Sale/SaleCard.vue [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+    </div>
+
+    <br />
+
+    <SaleInformation :currentSale="currentSale" />
+  </v-card>
+</template>
+
+<script setup>
+import SaleParticipantsInfo from "./SaleParticipantInfo.vue";
```
Comment (VladoKat) [inline src/components/Sale/SaleCard.vue]: This is still SaleParticipantsInfo. Please, refactor the name of the import as well as the file.

**Review by VladoKat — APPROVED** [review]

**Pull request merged**